tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -5899,29 +5919,34 @@ node_modules/chrome-devtools-frontend/front_end/cm/codemirror.js(9467,11): error
 node_modules/chrome-devtools-frontend/front_end/cm/codemirror.js(9468,11): error TS2339: Property 'wrapper' does not exist on type '{ cm: any; prevInput: string; pollingFast: boolean; polling: { id: any; set: (ms: any, f: any) =>...'.
 node_modules/chrome-devtools-frontend/front_end/cm/codemirror.js(9505,8): error TS2339: Property 'textarea' does not exist on type '{ cm: any; prevInput: string; pollingFast: boolean; polling: { id: any; set: (ms: any, f: any) =>...'.
 node_modules/chrome-devtools-frontend/front_end/cm/codemirror.js(9527,40): error TS2339: Property 'getValue' does not exist on type '{ options: any; doc: any; display: { input: any; }; state: { [x: string]: any; keyMaps: any[]; ov...'.
+node_modules/chrome-devtools-frontend/front_end/cm/comment.js(6,9): error TS2304: Cannot find name 'require'.
 node_modules/chrome-devtools-frontend/front_end/cm/comment.js(6,17): error TS2307: Cannot find module '../../lib/codemirror'.
 node_modules/chrome-devtools-frontend/front_end/cm/comment.js(7,19): error TS2304: Cannot find name 'define'.
 node_modules/chrome-devtools-frontend/front_end/cm/comment.js(7,43): error TS2304: Cannot find name 'define'.
 node_modules/chrome-devtools-frontend/front_end/cm/comment.js(8,5): error TS2304: Cannot find name 'define'.
+node_modules/chrome-devtools-frontend/front_end/cm/markselection.js(12,9): error TS2304: Cannot find name 'require'.
```

Comment:
should it have `'node'` in order to find `require`?

Reply:
`require` in these files is part of a UMD header. It's not actually in use, in the strictest sense:
```ts
(function(mod) {
  if (typeof exports == "object" && typeof module == "object") // CommonJS
    mod(require("../../lib/codemirror"));
  else if (typeof define == "function" && define.amd) // AMD
    define(["../../lib/codemirror"], mod);
  else // Plain browser env
    mod(CodeMirror);
})(function(CodeMirror) {
```
and chrome doesn't define it in their `externs` file (likely because the entire `cm` folder is marked as external, and so is not strictly checked).